extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -191,15 +230,9 @@ export class AmpAdApiHandler {
   }
 
   /** @override  */
```

Comment:
We can probably remove those override annotations in this class. I think we're not extending any class. Are we?